packages/beacon-node/src/chain/validation/attestation.ts

```diff
@@ -762,7 +763,7 @@ export function getCommitteeIndices(
 /**
  * Compute the correct subnet for a slot/committee index
  */
-export function computeSubnetForSlot(shuffling: EpochShuffling, slot: number, committeeIndex: number): number {
+export function computeSubnetForSlot(shuffling: EpochShuffling, slot: number, committeeIndex: number): SubnetID {
```

Comment:
Just realized `computeSubnetForSlot` here and `computeSubnetForCommitteesAtSlot` in `packages/beacon-node/src/api/impl/validator/utils.ts` can probably be just one function instead. Can combine them at a later PR